torch/quantization/quantize_script.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
you could have probably called the whole file `_quantize_script` then you don't need to rename individual functions.